src/main/java/org/isf/usergroups/dto/UserGroupDTO.java

```diff
@@ -31,36 +31,38 @@
 
 public class UserGroupDTO {
 
+	@Schema(description = "List of group's permissions")
+	List<PermissionDTO> permissions;
 	@NotNull
 	@Schema(description = "Name of the group (must be unique)", example = "labo", maxLength = 50)
 	private String code;
-
 	@Schema(description = "The description of the group", example = "Staff members working in the laboratory", maxLength = 128)
 	private String desc;
-
-	@Schema(description = "List of group's permissions")
-	List<PermissionDTO> permissions;
```

Comment:
why to switch order of this field in the code?

Reply:
It's the code style doing that ! I've just reformat the whole file using the IDE.

Reply:
I see... maybe because it is missing the `private` modifier so it fallsback to the default `protected` and therefore a different order? could you try please? 🤓